crowdloans/projects/kusama.json

```diff
@@ -338,6 +338,15 @@
         { "name": "Community DAO", "value": 10, "color": "#c0dfdd" },
         { "name": "Parachain Slot Reserve", "value": 5, "color": "#609c96" }
       ]
-    }
+  },  
+  {
+    "paraid": "2115",
+    "name": "DORA KSM Parachain",
+    "token": "DORA",
+    "description": "DoraFactory Kusama parachain",
+    "overview": "Dora Factory is the DAO-as-a-Service infrastructure that empowers decentralized organizations and open source ventures. The DORA KSM Parachain is Dora Factory’s Kusama parachain, which supports pallets like quadratic funding and multisig wallet. In connection with DoraHacks, the Dora Factory KSM Parachain will be able to support open source communities, hackathons, quadratic funding grants, and DAOs in general from the Kusama ecosystem.",
+    "website": "https://dorafactory.org",
+    "icon": "https://dorafacory-assets.s3.ap-southeast-1.amazonaws.com/dorafactory-logo.svg",
```

Comment:
could you add an icon in [our repo](https://github.com/parallel-finance/auction-toolkit/tree/main/icons/chains) and change the URL format to `https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/<image-name>`?

Reply:
> @tinydjp https://docs.parallel.fi/heiko-mainnet/crowdloan-v2-kusama-project-onboarding - pls see here on the information needed to onboard Dora. As soon as we get it, we can add.

We've just updated the lacked information, please  help review again, thanks!